src/mode/modeHandler.ts

```diff
@@ -1256,11 +1256,7 @@ export class ModeHandler implements vscode.Disposable {
       }
 
       if (!cachedMode.isVisualMode && cachedRegister !== RegisterMode.LineWise) {
-        if (Position.EarlierOf(start, stop) === start) {
```

Comment:
This is a bizarre if condition, considering that we set start to the earlier of [start, stop], unless start is equal to stop.